### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by anderCM**
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
  - If response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/controllers/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      expect(response).to render_template(:index)
+    end
+
+    it 'Should include Default text on template' do
+      expect(response.body).to include('List of posts here')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:post) { FactoryBot.create(:post) }
```
Comment (Davidosky007) [inline spec/controllers/posts_controller_spec.rb]: * According to the requirement you are to Create a Request spec file for all your controllers, kindly move all your tests into a request spec file as stated in the requirement 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @anderCM 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* All tests for controllers written 👍 
* All tests passed ✔️ 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/controllers/posts_controller_spec.rb (hunk 1 above)
Comment (anderCM) [reply spec/controllers/posts_controller_spec.rb]: Thanks @Davidosky007

**Review by andersonlebon — APPROVED** [review]
## Hi @anderCM 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**